### DISCUSSION TITLE
Vuetify3 Add Expansion Panels UI

### DISCUSSION THREAD

**Discussion by ghost** [Kitware/trame#549] [General]
Hello, I was wondering how do we implement this Vuetify expansion panels in Trame with Vue3. Is there any documentation for that? Any help will be appreciated :) 
Here is the link of an example for Vueitfy expansion panels: https://v15.vuetifyjs.com/en/components/expansion-panels/ 
![image](https://github.com/Kitware/trame/assets/93155074/5ef7f103-dac5-4e81-8cf7-b9b801161de1)

**Comment by jourdain** [comment]
https://vuetifyjs.com/en/components/expansion-panels/#usage

**Reply by ghost** [reply]
Do I just copy this section in Trame Python script? I'm very new to Trame and Vuetify :)

![Capture](https://github.com/Kitware/trame/assets/93155074/9516a4f2-9910-4871-86fa-ba48d1713ae9)

**Reply by jourdain** [reply]
Look at the slides shared [here in the doc](https://kitware.github.io/trame/guide/intro/course.html). In general, you should read the __How to start__ guide.
